commons/src/main/kotlin/com/simplemobiletools/commons/models/SimpleContact.kt

```diff
@@ -3,12 +3,38 @@ package com.simplemobiletools.commons.models
 import android.telephony.PhoneNumberUtils
 import com.simplemobiletools.commons.extensions.normalizePhoneNumber
 import com.simplemobiletools.commons.extensions.normalizeString
+import com.simplemobiletools.commons.helpers.SORT_BY_FULL_NAME
+import com.simplemobiletools.commons.helpers.SORT_DESCENDING
 
 data class SimpleContact(
     val rawId: Int, val contactId: Int, var name: String, var photoUri: String, var phoneNumbers: ArrayList<PhoneNumber>,
     var birthdays: ArrayList<String>, var anniversaries: ArrayList<String>
 ) : Comparable<SimpleContact> {
+
+    companion object {
+        var sorting = -1
+    }
+
     override fun compareTo(other: SimpleContact): Int {
+        if (sorting == -1) return compareByFullName(other)
+
+        var result = when {
+            sorting and SORT_BY_FULL_NAME != 0 -> {
+                compareByFullName(other)
```

Comment:
compact lines like this to 1 line please, like "sorting and SORT_BY_FULL_NAME != 0 -> compareByFullName(other)"